src/features/onboarding/AccountAssignScreen.tsx

```diff
@@ -66,23 +54,23 @@ const AccountAssignScreen = () => {
   )
 
   const allPaths = useMemo(() => {
-    if (
-      route.params?.secretKey &&
-      route.params?.derivationPath &&
-      !paths.some((p) => p.derivationPath === route.params?.derivationPath)
-    ) {
-      return [
-        ...paths,
-        {
-          keypair: Keypair.fromSecretKey(
-            Uint8Array.from(Buffer.from(route.params.secretKey, 'base64')),
-          ),
-          derivationPath: route.params.derivationPath,
-        },
-      ]
-    }
+    // if (
```

Comment:
Why is this commented out do we still need it?

Reply:
Not needed. I looked and I could not find anywhere these params were being passed